pkg/models/settings_test.go

```diff
@@ -118,6 +119,14 @@ func TestLoadSettings(t *testing.T) {
 				t.Errorf("LoadSettings() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+
+			assert.NotNil(t, gotSettings)
+
+			// settings.ProxyOpts are handled by the sdk, we just validate
+			assert.NotNil(t, gotSettings.ProxyOpts)
```

Comment:
This is a configuration that was added, but is built using the SDK. There is no easy way to test it (and in my opinion would be out of the scope of the plugin it self). So I decided to just skip the validation of it. Another approach would be to build  a struct with the default values, but I don't think it will add much value